qdrant_client/local/local_collection.py

```diff
@@ -401,7 +401,7 @@ def search(
             scores = calculate_context_scores(query_vector, vectors[: len(self.payload)], distance)
         elif isinstance(query_vector, SparseVector):
             # sparse vector query must be sorted by indices for dot product to work with persisted vectors
-            sort(query_vector)
+            query_vector = sort_sparse_vector(query_vector)
```

Comment:
That's embarrassing, I guess I thought this was a sort in place :face_with_spiral_eyes:

Reply:
🙈